x/epochs/keeper/epoch_infos.go

```diff
@@ -9,9 +12,28 @@ import (
 
 // GetEpochInfo returns epoch info by identifier
 func (k Keeper) GetEpochInfo(ctx sdk.Context, identifier string) (types.EpochInfo, bool) {
+	duration, found := k.GetEpochDuration(ctx, identifier)
+	if !found {
+		return types.EpochInfo{}, false
+	}
+	return k.GetEpoch(ctx, duration)
+}
+
+// GetEpochDuration returns epoch duration by identifier
+func (k Keeper) GetEpochDuration(ctx sdk.Context, identifier string) ([]byte, bool) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixEpochDuration)
+	bz := store.Get([]byte(identifier))
+	if len(bz) == 0 {
+		return make([]byte, 0), false
+	}
+	return bz, true
+}
+
+// GetEpochInfo returns epoch info by duration
+func (k Keeper) GetEpoch(ctx sdk.Context, duration []byte) (types.EpochInfo, bool) {
```

Comment:
@loredanacirstea what do you think of renaming `EpochInfo` to `Epoch` in the proto files? This will make the code less wordy and I don't see the benefit in have `Info` added.